SemDiff.Core/GitHub.cs

```diff
@@ -173,6 +217,23 @@ private static string ParseNextLink(IEnumerable<string> links)
             {
                 return null;
             }
+            if (CurrentSaved != null)
+            {
+                var removePRs = CurrentSaved;
+                foreach (var pr in pullRequests)
+                {
+                    foreach (var prRemove in removePRs)
+                    {
+                        if (pr.Number == prRemove.Number)
+                        {
```

Comment:
Need to add a line here like `pr.LastWrite = prRemove.LastWrite` to preserve the state in the new object